### DISCUSSION TITLE
Fooocus download folder now showing up

### DISCUSSION THREAD

**Discussion by QT36** [lllyasviel/Fooocus#3179] [Q&A] [answered]
Hi all,

When I click download Fooocus downloads but the zip folder that I need to extract doesn't show up. The only thing that shows up is a file with a sheet of paper as the thumbnail. Can someone help me?
Thanks

**Comment by ILikeToasters** [comment, marked as answer]
Are you talking about the file to install Fooocus? That should be a file named Fooocus_win64_2-1-831.7z which you will probably need to install 7zip program to extract it. It is not a standard zip file and uses the 7zip format which is better. You can get it here: https://7-zip.org/
